sentry-android-core/src/main/java/io/sentry/android/core/PerformanceAndroidEventProcessor.java

```diff
@@ -32,6 +33,22 @@
         Objects.requireNonNull(activityFramesTracker, "ActivityFramesTracker is required");
   }
 
+  /**
+   * Returns the event itself
+   *
+   * @param event the SentryEvent the SentryEvent
+   * @param hint the Hint the Hint
```

Comment:
confusing 🤪

Reply:
indeed, it was auto-generated, didn't realize the confusion :(